src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -250,11 +254,21 @@ public void downloadZippedFile(Environment env, HttpServletRequest request,
 				if (COLLECTION_TYPE.equals(type)) {
 					// folders
 					if (rootFolderPath == null) {
-						rootFolderPath = contentUrl;
+						rootFolderPath = contentUrl.substring(0, contentUrl.length());
 					} else {
 						// create the zipentry for the sub-folder first
 						String folderName = contentUrl.replace(rootFolderPath,
 								"");
+						// update folder name
+						folderNameMap = Utils.updateFolderName(folderNameMap, title,
```

Comment:
right. Title is the title shown in CTools UI, vs folderName is the substring from access url.